src/App.js

```diff
@@ -28,7 +30,6 @@ const App = () => {
   const fetchProductData = async () => {
```

Comment:
API call should be inside ProductList component. Why are we fetching users and products in app.